libraries/botbuilder-dialogs-adaptive-testing/src/testRunner.ts

```diff
@@ -16,34 +16,33 @@ import { TestScript } from './testScript';
  * In charge of running Dialog Adaptive tests.
  */
 export class TestRunner {
-    private resourceExplorer: ResourceExplorer;
-    private testAdapter: TestAdapter;
-
     /**
      * Initializes a new instance of the `TestRunner` class.
      * @param resourcePath Path where the `.dialog` files are located.
      */
-    public constructor(resourcePath: string) {
+    public constructor(private resourcePath: string) {
         ComponentRegistration.add(new AdaptiveComponentRegistration());
         ComponentRegistration.add(new AdaptiveTestComponentRegistration());
         ComponentRegistration.add(new QnAMakerComponentRegistration());
         ComponentRegistration.add(new LuisComponentRegistration());
-
-        this.resourceExplorer = new ResourceExplorer();
-        this.resourceExplorer.addFolder(resourcePath, true, false);
-
-        this.testAdapter = new TestAdapter(TestAdapter.createConversation('botbuilder-dialogs-adaptive-testing'));
     }
 
     /**
      * Runs a test script with the specified name.
      * @param testName Test name.
      * @returns A Promise that represents the work queued to execute.
      */
-    public async runTestScript(testName: string): Promise<any> {
-        const script = this.resourceExplorer.loadType<TestScript>(`${testName}.test.dialog`);
+    public runTestScript(testName: string): Promise<void> {
+        // Construct a fresh test adapter
+        const testAdapter = new TestAdapter(TestAdapter.createConversation('botbuilder-dialogs-adaptive-testing'));
```

Comment:
I'm unfamiliar with `botbuilder-dialogs-adaptive-testing`... Should the `TestRunner` always be responsible for the TestAdapter?

Developers can't pass in their own `TestAdapter` and are bound to the test arrangement we've setup for them. I'm not sure if passing in their own TestAdapter is a possible scenario, but it's worth considering...

Reply:
I wasn't able to find any instances where, currently, a test file contains both `TestAdapter` and `TestRunner`. Not sure if there's a scenario where a dev would need to pass in their own `TestAdapter`, either, but that's more due to lack of familiarity with the library